src/Akeneo/UserManagement/Bundle/Controller/Rest/UserGroupController.php

```diff
@@ -45,4 +147,21 @@ public function indexAction()
 
         return new JsonResponse($userGroups);
     }
+
+    /**
+     * @param $identifier
+     * @return Group
+     */
+    private function getUserGroupOr404($identifier):Group
```

Comment:
```suggestion
    private function getUserGroupOr404($identifier): Group
```

and doc block to remove